src/gui/debuggingdialog.h

```diff
@@ -63,6 +63,7 @@ class DebuggingDialog : public CustomDialog {
         bool _deleteLogs = false;
         bool _needToSave = false;
         bool _sendArchivedLogs = true;
+        QString _heavyLogLabelStr;
```

Comment:
Would `static const` here  cause a new problem? What was the cause of the original problem?

Reply:
Using a `static const`, means the string gets initialized when the app starts up. At that moment, `QTranslator` sets `heavyLogLabelStr` to English text because there's no language set yet. This leads to the text staying in English, even if you pick another language, because cause it is not being updated after that.

To solve this, we can call `QObject::tr` when we open the modal. 
At best, I can make it const and initialize it in the constructor, but using static const isn't working in this case.